### PR TITLE
adding a bioconductor base

### PR DESCRIPTION
**Pull request by emmjab**
an alternative to starting with the `rocker` docker image for R. 

test from image built here: https://hub.docker.com/r/emmjab/r-bioconductor/tags
and project here: https://renkulab.io/projects/emma.jablonski/rbasic/files

### CODE REVIEW

**Comment by rokroskar** [comment]
What's the status here @emmjab and @cchoirat?

**Comment by cchoirat** [comment]
@emmjab, not sure how r-bioconductor gets created.

**Comment by emmjab** [comment]
The idea from this was to use the bioconductor project as a base and add the renku dependencies on top, to share with the bioconductor community. I haven't looked at this in a while so i'll see if there were any bioconductor version updates and re-test.

**Comment by emmjab** [comment]
The latest commits with the symlink & project opening for r projects (both the base and the bioconductor) "work", but also add in the project contents into the parent directory... need to fix this.

File: docker/r-bioconductor_3_10/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+USER ${NB_USER}
+
+# set up conda in the NB_USER environment
+RUN echo ". ${CONDA_PATH}/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    echo "conda activate base" >> ~/.bashrc
+
+# install the jupyter stack
+ARG JUPYTERHUB_VERSION=0.9.6
+#RUN conda install --quiet --yes \
```
Comment (rokroskar) [inline docker/r-bioconductor_3_10/Dockerfile]: can we remove the commented pieces if they are not providing documentation?

File: docker/r-bioconductor_3_10/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+RUN ${CONDA_PATH}/bin/conda install --quiet --yes conda-build
+RUN ${CONDA_PATH}/bin/conda build purge-all
+RUN npm cache clean --force
+
+RUN jupyter notebook --generate-config && \
+    rm -rf ${CONDA_PATH}/share/jupyter/lab/staging && \
+    rm -rf /home/$NB_USER/.cache/yarn && \
+    pip install -U --no-cache-dir pip wheel
+
+#RUN pip install --no-cache-dir jupyter-rsession-proxy
```
Comment (rokroskar) [inline docker/r-bioconductor_3_10/Dockerfile]: same here - remove commented pieces

**Pull request merged**